src/Symfony/Bundle/DebugBundle/DependencyInjection/DebugExtension.php

```diff
@@ -65,13 +65,9 @@ public function load(array $configs, ContainerBuilder $container)
             $container->getDefinition('data_collector.dump')
                 ->replaceArgument(4, new Reference('var_dumper.cli_dumper'))
             ;
-        }
-
-        if (!isset($serverDumperHost)) {
-            $container->getDefinition('var_dumper.command.server_dump')->setClass(ServerDumpPlaceholderCommand::class);
-            if (!class_exists(ServerDumper::class)) {
```

Comment:
this check is not needed since VarDumper is a mandatory requirement of DebugBundle